README.md

```diff
@@ -11,10 +11,12 @@
 1. To set-up the environment for you'll need:
 
 - [Ruby](https://www.ruby-lang.org/en/downloads/) (2.4 or above)
+- [Python 2](https://www.python.org/downloads/)
```

Comment:
Something in `npm install` needs Python 2. Don't know what exactly, but it definitely fails if Python 2 is not present.